metadata_harvester_cli.py

```diff
@@ -1,30 +1,80 @@
 """
-Command line interface for the metadata harvester
+Main script for running metadata harvesting and sending it to Metax.
 """
 
-import click
-
 from harvester.pmh_interface import PMH_API
 from harvester.metadata_parser import MSRecordParser
+import metax_api
 from lxml import etree
+import logging
+
+logger_harvester = logging.getLogger("harvester")
+logger_harvester.setLevel(logging.DEBUG)
+file_handler_harvester = logging.FileHandler("harvester.log")
+file_handler_harvester.setFormatter(logging.Formatter("%(asctime)s - %(levelname)s - %(message)s"))
+logger_harvester.addHandler(file_handler_harvester)
+
+def get_last_harvest_date():
+    """Only successful harvests are logged. This function gets the last harvesting date from the log.
+    :param filename: string value of a file name
+    :return: date in last line
+    """
+    log_file_path = "harvester.log"
+    with open(log_file_path, "r") as file:
```

Comment:
If retrieve_metadata_content() doesn't find the file, it will move on to fetching all available records from the api. We can raise the error if you think it'd be better that way.

Reply:
Implemented the FileNotFoundError because during testing, the code couldn't progress without handling the error correctly.